### PR TITLE
Specify Editor or Admin (non-viewer) as team members who have access to premium features.

### PR DESCRIPTION
**Pull request by kevinmaes**

### CODE REVIEW

File: docs/studio-pro-plan.mdx [hunk 1]
```diff
@@ -44,7 +44,7 @@ You can choose between monthly or yearly billing on all of our paid plans.
 
 ### Pro features as a team member
 
-If you are a team member of another user’s [team](teams.mdx), you also have access to [premium features](#pro-features)! However, if you want to create your own teams, you need to [upgrade to your own Team subscription](upgrade.mdx).
+If you are a team member of another user’s [team](teams.mdx), with the role of [Admin](teams.mdx#admin-role) or [Editor](teams.mdx#editor-role), you also have access to [premium features](#pro-features)! However, if you want to create your own teams, you need to [upgrade to your own Team subscription](upgrade.mdx).
```
Comment (laurakalbag) [inline docs/studio-pro-plan.mdx:47]:
```suggestion
If you are a team member of another user’s [team](teams.mdx), with the role of [Admin](/docs/teams/#admin-role) or [Editor](/docs/teams/#editor-role), you also have access to [premium features](#pro-features)! However, if you want to create your own teams, you need to [upgrade to your own Team subscription](upgrade.mdx).
```

Docusaurus can struggle to resolve heading links that use the `.mdx` extension, so it’s easier to make them relative like this. (Yes, it’s incredibly annoying. I’m trying to work on some docs docs to help explain this better!)

File: docs/studio-team-plan.mdx [hunk 2]
```diff
@@ -34,7 +34,7 @@ You can choose between monthly or yearly billing on all of our paid plans.
 
 ### Pro features as a team member
 
-If you are a team member of another user’s [team](teams.mdx), you also have access to [premium features](studio-pro-plan.mdx)! However, if you want to create your own teams, you need to [upgrade to your own Pro or Team subscription](upgrade.mdx).
+If you are a team member of another user’s [team](teams.mdx), with the role of [Admin](teams.mdx#admin-role) or [Editor](teams.mdx#editor-role), you also have access to [premium features](studio-pro-plan.mdx)! However, if you want to create your own teams, you need to [upgrade to your own Pro or Team subscription](upgrade.mdx).
```
Comment (laurakalbag) [inline docs/studio-team-plan.mdx:37]:
```suggestion
If you are a team member of another user’s [team](teams.mdx), with the role of [Admin](/docs/teams/#admin-role) or [Editor](/docs/teams/#editor-role), you also have access to [premium features](studio-pro-plan.mdx)! However, if you want to create your own teams, you need to [upgrade to your own Pro or Team subscription](upgrade.mdx).
```

File: versioned_docs/version-4/studio-pro-plan.mdx [hunk 3]
```diff
@@ -44,7 +44,7 @@ You can choose between monthly or yearly billing on all of our paid plans.
 
 ### Pro features as a team member
 
-If you are a team member of another user’s [team](teams.mdx), you also have access to [premium features](studio-pro-plan.mdx)! However, if you want to create your own teams, you need to [upgrade to your own Team subscription](/upgrade.mdx).
+If you are a team member of another user’s [team](teams.mdx), with the role of [Admin](teams.mdx#admin-role) or [Editor](teams.mdx#editor-role), you also have access to [premium features](studio-pro-plan.mdx)! However, if you want to create your own teams, you need to [upgrade to your own Team subscription](/upgrade.mdx).
```
Comment (laurakalbag) [inline versioned_docs/version-4/studio-pro-plan.mdx:47]:
```suggestion
If you are a team member of another user’s [team](teams.mdx), with the role of [Admin](/docs/teams/#admin-role) or [Editor](/docs/teams/#editor-role), you also have access to [premium features](studio-pro-plan.mdx)! However, if you want to create your own teams, you need to [upgrade to your own Team subscription](/upgrade.mdx).
```

File: versioned_docs/version-4/studio-team-plan.mdx [hunk 4]
```diff
@@ -34,7 +34,7 @@ You can choose between monthly or yearly billing on all of our paid plans.
 
 ### Pro features as a team member
 
-If you are a team member of another user’s [team](teams.mdx), you also have access to [premium features](/docs/studio-pro-plan.mdx)! However, if you want to create your own teams, you need to [upgrade to your own Pro or Team subscription](upgrade.mdx).
+If you are a team member of another user’s [team](teams.mdx), with the role of [Admin](teams.mdx#admin-role) or [Editor](teams.mdx#editor-role), you also have access to [premium features](/docs/studio-pro-plan.mdx)! However, if you want to create your own teams, you need to [upgrade to your own Pro or Team subscription](upgrade.mdx).
```
Comment (laurakalbag) [inline versioned_docs/version-4/studio-team-plan.mdx:37]:
```suggestion
If you are a team member of another user’s [team](teams.mdx), with the role of [Admin](/docs/teams/#admin-role) or [Editor](t/docs/teams/#editor-role), you also have access to [premium features](/docs/studio-pro-plan.mdx)! However, if you want to create your own teams, you need to [upgrade to your own Pro or Team subscription](upgrade.mdx).
```

File: docs/studio-pro-plan.mdx (hunk 1 above)
Comment (kevinmaes) [reply docs/studio-pro-plan.mdx:47]: I really didn't understand when to use one method over another. Thanks for explaining that!

**Pull request merged**